Xamarin.Forms.Platform.Tizen/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.Shapes;
+using SkiaSharp;
+using FormsRectangle = Xamarin.Forms.Rectangle;
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+    public static class GeometryExtensions
+    {
+        public static SKPath ToSKPath(this Geometry geometry)
+        {
+            SKPath path = new SKPath();
+
+            if (geometry is LineGeometry)
+            {
+                path = MakePath(geometry as LineGeometry);
+            }
+            else if (geometry is RectangleGeometry)
+            {
+                path = MakePath(geometry as RectangleGeometry);
+            }
+            else if (geometry is EllipseGeometry)
+            {
+                path = MakePath(geometry as EllipseGeometry);
+            }
+            else if (geometry is GeometryGroup)
+            {
+                path = MakePath(geometry as GeometryGroup);
+            }
+            else if (geometry is PathGeometry)
+            {
+                path = MakePath(geometry as PathGeometry);
+            }
```

Comment:
You can call MakePath method without casing. 
```suggestion
  path = MakePath(geometry)
```

Reply:
Updated using `dynamic` cast.